ccd-definition/CaseField/CareSupervision/hearingOrders.json

```diff
@@ -157,5 +157,64 @@
     "Label": "Which hearing does this CMO relate to?",
     "FieldType": "DynamicList",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingOrderDraftInfo",
+    "Label": "<div class='panel panel-border-wide'>You can add CMOs and C21s that relates to the same hearing in one go.<br/><br/>If they are for different hearings, or are not related to any hearings, add these separately.</div>",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingOrderDraftKind",
+    "Label": "What order are you adding?",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "DraftOrderKind",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingsForHearingOrderDrafts",
+    "Label": "Which hearing does the order relates to?",
+    "HintText": "Choose 'No hearing' if it does not relate to hearing",
+    "FieldType": "DynamicList",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "currentHearingOrderDrafts",
+    "Label": "Order",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "UploadedCaseManagementOrder",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingOrdersBundlesDrafts",
+    "Label": "Hearing",
```

Comment:
Not sure hearing is the right label here. although I am not sure what it should be. Has this been confirmed by UX?

Reply:
Also something that this collection points out, we probs never want to have an object that has a complex type depth of more than 3 (this one is main collection ->  main object -> supporting docs collection), on my laptop things are becoming very cramped (not sure about larger screen sizes)

Reply:
beautiful :D 

![image](https://user-images.githubusercontent.com/22620804/105499912-0efbfd80-5cba-11eb-8932-b80c40f30294.png)

Reply:
both thinks requested by xui